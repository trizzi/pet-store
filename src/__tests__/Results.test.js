/**
 * @jest-environment jsdom
 */

import { expect, test } from "@jest/globals";
import { create } from "react-test-renderer";
import Results from "../Results";

test("renders correctly with no pets", async () => {
  const tree = create(<Results pets={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
