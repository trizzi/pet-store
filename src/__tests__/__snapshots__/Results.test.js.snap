// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with no pets 1`] = `
<div
  className="grid gap-4 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 py-4"
>
  <h1>
    No Pets Found
  </h1>
</div>
`;
